src/tito/builder/__init__.py

```diff
@@ -15,4 +15,8 @@
     GitLfsBuilder, \
     MeadBuilder
 
+from tito.builder.submodule_aware_builder import \
+    SubmoduleAwareBuilder
+
 from tito.builder.fetch import FetchBuilder
+
```

Comment:
The CI tests fail because they do a PEP8 check and it complains about this empty line. Can you please drop it?

Reply:
Fixed.  I think I ended up with an extra newline when shipping patch files around